commands.go

```diff
@@ -325,8 +325,21 @@ func cmdCreate(c *cli.Context) {
 		log.Fatalf("error setting active host: %v", err)
 	}
 
+	info := ""
+	userShell := filepath.Base(os.Getenv("SHELL"))
+
+	switch userShell {
+	case "fish":
+		info = fmt.Sprintf("%s env %s | source", c.App.Name, name)
+	default:
+		info = fmt.Sprintf("$(%s env %s)", c.App.Name, name)
+	}
+
 	log.Infof("%q has been created and is now the active machine.", name)
-	log.Infof("To point your Docker client at it, run this in your shell: $(%s env %s)", c.App.Name, name)
+
+	if info != "" {
```

Comment:
This if statement probably isn't needed, the default from the switch would cover it